test/seeds/recipe-creation.js

```diff
@@ -53,3 +53,8 @@ module.exports = async () => {
     created_at: '2021-10-10',
   })
 }
+
+module.exports = {
+  cleanup,
```

Comment:
changed because I needed to use`cleanup` in `after()`. DB data from one group of tests was breaking other groups of tests (foreign key constraints stopping deletes)